packages/nextjs/src/config/withSentryConfig.ts

```diff
@@ -107,17 +107,17 @@ function getFinalConfigObject(
         showedExportModeTunnelWarning = true;
         // eslint-disable-next-line no-console
         console.warn(
-          '[@sentry/nextjs] The Sentry Next.js SDK `tunnelRoute` option will not work in combination with Next.js static exports. The `tunnelRoute` option uses serverside features that cannot be accessed in export mode. If you still want to tunnel Sentry events, set up your own tunnel: https://docs.sentry.io/platforms/javascript/troubleshooting/#using-the-tunnel-option',
+          '[@sentry/nextjs] The Sentry Next.js SDK `tunnelRoute` option will not work in combination with Next.js static exports. The `tunnelRoute` option uses server-side features that cannot be accessed in export mode. If you still want to tunnel Sentry events, set up your own tunnel: https://docs.sentry.io/platforms/javascript/troubleshooting/#using-the-tunnel-option',
         );
       }
     } else {
       const resolvedTunnelRoute =
-        typeof userSentryOptions.tunnelRoute === 'boolean'
+        typeof userSentryOptions.tunnelRoute === 'boolean' && userSentryOptions.tunnelRoute === true
```

Comment:
We could simplify that, since we know it's a boolean once we pass ` typeof userSentryOptions.tunnelRoute === 'boolean'`
```suggestion
        typeof userSentryOptions.tunnelRoute === 'boolean' && userSentryOptions.tunnelRoute
```

Perhaps a bit redundant otherwise